types/cloudinary_ts_spec.ts

```diff
@@ -568,6 +568,42 @@ cloudinary.v2.api.usage(function (error, result) {
 // $ExpectType Promise<any>
 cloudinary.v2.api.usage({public_id: 'demo'});
 
+cloudinary.v2.api.add_metadata_field({
```

Comment:
@strausr , isn't the $ExpectType comment important here?

Reply:
No. We can't check the type of the promise bc call_api is not really called and will always return nothing. Its enough to verify the promise with .then() and a sanity check for us that the functions are accepting the proper fields